fixtures/babel-component/src/index.jsx

```diff
@@ -4,7 +4,7 @@ const Button = styled.button`
   color: blue;
   font-size: 30px;
   border: 2px solid blue;
-  padding: 8px;
+  padding: 32px;
```

Comment:
`32px` will be changed to `2pc` if optimization is enabled. I changed the value for testing